### PR TITLE
todoist: add page

### PR DESCRIPTION
**Pull request by iuliandita**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/todoist.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# todoist
+
+> This program will let you use Todoist from the CLI.
```
Comment (CleanMachine1) [inline pages/common/todoist.md:3]:
```suggestion
> Access Todoist from the command-line.
```

File: pages/common/todoist.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://github.com/sachaos/todoist>.
+
+- Add a task:
+
+`todoist add "{{some_task}}"`
+
+- Add a high priority task, with label, project, and due date:
```
Comment (CleanMachine1) [inline pages/common/todoist.md:10]:
```suggestion
- Add a high priority task with a label, project, and due date:
```

File: pages/common/todoist.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# todoist
+
+> This program will let you use Todoist from the CLI.
+> More information: <https://github.com/sachaos/todoist>.
+
+- Add a task:
+
+`todoist add "{{some_task}}"`
```
Comment (CleanMachine1) [inline pages/common/todoist.md:8]:
```suggestion
`todoist add "{{task_name}}"`
```

File: pages/common/todoist.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Add a high priority task, with label, project, and due date:
+
+`todoist add "{{some_task}}" --priority {{1}} --label-ids "{{label_id}}" --project-name "{{project_name}}" --date "{{tmr 9am}}"`
```
Comment (CleanMachine1) [inline pages/common/todoist.md:12]:
```suggestion
`todoist add "{{task_name}}" --priority {{1}} --label-ids "{{label_id}}" --project-name "{{project_name}}" --date "{{tmr 9am}}"`
```

for the date, can "{{tomorrow 9am}}" work?

File: pages/common/todoist.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Add the same task, in quick mode:
+
+`todoist quick '#{{project_name}} "{{tmr 9am}}" p{{1}} {{some_task}} @{{label_name}}'`
```
Comment (CleanMachine1) [inline pages/common/todoist.md:16]:
```suggestion
`todoist quick '#{{project_name}} "{{tmr 9am}}" p{{1}} {{task_name}} @{{label_name}}'`
```

Should there be a space between the @

File: pages/common/todoist.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Add a task:
+
+`todoist add "{{some_task}}"`
+
+- Add a high priority task, with label, project, and due date:
+
+`todoist add "{{some_task}}" --priority {{1}} --label-ids "{{label_id}}" --project-name "{{project_name}}" --date "{{tmr 9am}}"`
+
+- Add the same task, in quick mode:
```
Comment (CleanMachine1) [inline pages/common/todoist.md:14]: Duplicating?

File: pages/common/todoist.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`todoist quick '#{{project_name}} "{{tmr 9am}}" p{{1}} {{some_task}} @{{label_name}}'`
+
+- List all tasks, with header and pretty colors:
```
Comment (CleanMachine1) [inline pages/common/todoist.md:18]:
```suggestion
- List all tasks with a header and color:
```

File: pages/common/todoist.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+`todoist --header --color list`
+
+- List all high priority tasks:
+
+`todoist --header --color list --filter p{{1}}`
+
+- List today's tasks with high priority that have the label `{{label_name}}`:
```
Comment (CleanMachine1) [inline pages/common/todoist.md:26]:
```suggestion
- List today's tasks with high priority that have the a specified label:
```

**Review by CleanMachine1 — APPROVED** [review]

**Comment by CleanMachine1** [comment]
You seemed to have messed up the CLA, maybe rebase and change the author of the "contributor" that isn't your GH

File: pages/common/todoist.md (hunk 4 above)
Comment (iuliandita) [reply pages/common/todoist.md:12]: yes, that would work too, the todoist api uses natural language

File: pages/common/todoist.md (hunk 5 above)
Comment (iuliandita) [reply pages/common/todoist.md:16]: no, the label has to be specified like "@labelname"

File: pages/common/todoist.md (hunk 6 above)
Comment (iuliandita) [reply pages/common/todoist.md:14]: the `quick` command calls the todoist api directly, this method uses the todoist-cli commands and options

**Comment by iuliandita** [comment]
Indeed I did, my bad. I did a rebase for ad36bd0 and then amended the author, I'm not sure if this fixed it...

**Comment by CleanMachine1** [comment]
I have no idea what you have done here.

Heres my suggestion, commit the changes requested, close the PR, copy the page with the edits, create a new git branch on your fork, ...... (make sure to not mess up the git credentials by using a different email)

**Comment by iuliandita** [comment]
Yeah, I have no idea what happened, I will create a new PR with your suggestions. Thanks.

**Comment by iuliandita** [comment]
@CleanMachine1 could you please submit a request to GitHub support to delete this PR?
I accidentally left some info in there that should not be on GitHub, and I also removed the project I submitted my PR from...
I contacted them already, but they said that the request should come from an admin here.
You could mention ticket #1272715.
Thanks a lot!

**Comment by marchersimon** [comment]
I contacted them just now. I hope it works.

**Pull request closed without merging**